README.md

```diff
@@ -3871,3 +3871,9 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 We encourage you to fork this guide and change the rules to fit your team’s style guide. Below, you may list some amendments to the style guide. This allows you to periodically update your style guide without having to deal with merge conflicts.
 
 # };
+
+
+## Additional Links and References for Learning more about Javascript
+
+1. https://www.codecademy.com/learn/learn-javascript
+2. https://www.w3schools.com/js/
```

Comment:
w3schools is terrible, and nothing should ever link to it, and nobody should ever read it - please remove this one.
```suggestion
```